packages/dev/core/src/Rendering/IBLShadows/iblShadowsRenderPipeline.ts

```diff
@@ -1088,6 +1068,10 @@ export class IblShadowsRenderPipeline extends PostProcessRenderPipeline {
      * Disposes the IBL shadow pipeline and associated resources
      */
     public override dispose() {
+        const materials = this._materialsWithRenderPlugin.splice(0);
+        materials.forEach((mat) => {
+            this.removeShadowReceivingMaterial(mat);
+        });

```

Comment:
You should also dispose the dummy textures.

Reply:
Done